docs/API.md

```diff
@@ -276,7 +276,7 @@ Parameters.
 Each token takes the following fields for it's create method respectively:
 
 * sessionToken : data, uid, createdAt, uaBrowser, uaBrowserVersion, uaOS, uaOSVersion, uaDeviceType,
-                 tokenVerificationId
+                 mustVerify, tokenVerificationId
```

Comment:
Circa line 329, `mustVerify` also needs adding to the documented returned properties of `sessionTokenWithVerificationStatus`.